src/dyld/darling

```diff
@@ -66,8 +66,19 @@ setup_prefix() {
     rm -f "$1/dev"
     ln -sf "system-root/dev" "$1/dev"
 
+    rm -rf "$1/sandboxed-home"
+
     rm -f "$1/home"
-    ln -sf "system-root/home" "$1/home"
+    if [ "${DARLING_SANDBOXED_HOME}" != "1" ]; then
+        >&2 echo "Setting sandboxed home"
+        mkdir "$1/sandboxed-home"
+        ln -sf "sandboxed-home" "$1/home"
+        for USERDIR in $(ls "$1/system-root/home/"); do
```

Comment:
Shouldn't this be `"/system-root/home/"` and not `"$1/system-root/home/"`?
Also, this will cause troubles if there are directories containing space in name.
So I'm sure this should be:
  `for USERDIR in /system-root/home/*; do`